build/config.props

```diff
@@ -31,9 +31,16 @@
     <TestSDKVersion>15.5.0</TestSDKVersion>
     <MoqVersion>4.10.1</MoqVersion>
     <FluentAssertionsVersion>4.19.4</FluentAssertionsVersion>
-    <MicrosoftBuildPackageVersion>15.1.1012</MicrosoftBuildPackageVersion>
+
+    <MicrosoftBuildFrameworkPackageVersion>16.0.461</MicrosoftBuildFrameworkPackageVersion>
+    <MicrosoftBuildTasksCorePackageVersion>16.0.461</MicrosoftBuildTasksCorePackageVersion>
+    <MicrosoftBuildUtilitiesCorePackageVersion>16.0.461</MicrosoftBuildUtilitiesCorePackageVersion>
+
   </PropertyGroup>
 
+  <!-- Special project import for dotnet source build -->
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
```

Comment:
Can you elaborate more on why we are doing this.

Reply:
That is the implementation of the Repo API for dotnet source-build capabilities.
In that import, the dotnet source-build injects a props file with source-builded package versions available at build time.